src/components/FileTree/index.spec.tsx

```diff
@@ -112,27 +105,24 @@ describe(__filename, () => {
       const fileName = 'same-nfile';
 
       const entries = [
-        {
-          depth: 0,
-          type: VersionEntryType.directory,
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
+          directory: true,
           filename: directoryName,
-          path: directoryName,
-          modified: '2019-01-01',
-        },
-        {
+        }),
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
+          directory: true,
           depth: 1,
-          type: VersionEntryType.directory,
           filename: directoryName,
           path: `${directoryName}/${directoryName}`,
-          modified: '2019-01-01',
-        },
-        {
+        }),
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
           depth: 2,
-          type: VersionEntryType.text,
```

Comment:
As above, maybe include `directory: false` to be explicit?

Reply:
:+1: